insights/parsers/ls_etc.py

```diff
@@ -71,3 +75,8 @@
 class LsEtc(FileListing):
     """Parses output of ``ls -lanR /etc`` command."""
     pass
+
+
+if __name__ == '__main__':
```

Comment:
I would strongly discourage this usage as very few people run the doctests manually.  Instead, one can test the documentation using an explicit test, e.g.:
```python
def test_ls_etc_documentation():
    failed, total = doctest.testmod(ls_etc, globs={
        'ls_etc': LsEtc(context_wrap(LS_ETC_DOCUMENTATION))
    })
    assert failed == 0
```
When the tests are run, if any of the example code fails then the user will be alerted.